src/stylesheets/app.scss

```diff
@@ -1,115 +1,155 @@
-    border-top-right-radius: 15px;
-    padding: 0.9375rem 1.875rem 0.6875rem 1.875rem;
-    position: relative;
-    p {
-        color: white;
-        font-size: 1.5rem;
-        font-weight: 400;
-        margin: 0;
-        text-transform: uppercase;
-        width: 87.5%;
-    }
-    img {
-        position: absolute;
-        right: 21px;
-        top: 22px;
-    }
+  background-color: #a15693;
+  border-top-left-radius: 15px;
+  border-top-right-radius: 15px;
+  padding: 0.9375rem 1.875rem 0.6875rem 1.875rem;
+  position: relative;
+
+  p {
+    color: white;
+    font-size: 1.5rem;
+    font-weight: 400;
+    margin: 0;
+    text-transform: uppercase;
+    width: 87.5%;
+  }
+
+  img {
+    position: absolute;
+    right: 21px;
+    top: 22px;
+  }
 }
 
 nav {
-    &:after {
-        clear: both;
-        content: "";
-        display: block;
-    }
-    ul {
-        float: right;
-    }
-    li {
-        list-style: none;
-        a {
-            color: white;
-            display: inline-block;
-            font-weight: 400;
-            list-style: none;
-            padding: 1rem 0.8rem 0.8rem 0.5rem;
-        }
+
+  &:after {
+    clear: both;
+    content: "";
+    display: block;
+  }
+
+  ul {
+    float: right;
+  }
+
+  li {
+    list-style: none;
+    
+    a {
+      color: white;
+      display: inline-block;
+      font-weight: 400;
+      list-style: none;
+      padding: 1rem 0.8rem 0.8rem 0.5rem;
     }
+  }
 }
 
 .chatbox {
-    background-color: white;
-    min-height: 605px;
-    padding: 1.1875rem 0.75rem;
-    &:after {
-        clear: both;
-        content: "";
-        display: block;
-    }
+  background-color: white;
+  min-height: 605px; // the height of the nav when open
+  max-height: 605px; // the max-height of the nav opens
+  padding: 1.1875rem 0.75rem;
+  overflow: scroll;
+  &:after {
+    clear: both;
+    content: "";
+    display: block;
+  }
 }
 
 .message {
-    border-radius: 4px;
-    clear: both;
-    font-size: 1rem;
-    font-weight: 300;
-    max-width: 80.83333333%;
-    padding: 0.9375rem 0.9375rem 0.9375rem 1rem;
-    p {
-        margin: 0;
-    }
-    &.message-left {
-        background-color: #F7F4E4;
-        float: left;
-        margin-bottom: 0.75rem;
-    }
-    &.message-right {
-        background-color: #D7F0F3;
-        float: right;
-        margin-bottom: 0.9rem;
-    }
+  animation: msgSlideUp 0.3s both ease-in-out;
+  border-radius: 4px;
+  clear: both;
+  font-size: 1rem;
+  font-weight: 300;
+  max-width: 80.83333333%;
+  padding: 0.9375rem 0.9375rem 0.9375rem 1rem;
+  
+  p {
+    margin: 0;
+  }
+
+  &.message-left {
+    background-color: #F7F4E4;
+    float: left;
+    margin-bottom: 0.75rem;
+  }
+
+  &.message-right {
+    background-color: #D7F0F3;
+    float: right;
+    margin-bottom: 0.9rem;
+  }
 }
 
+@keyframes msgSlideUp {
+  from {
+    opacity: 0;
+    transform: translateY(10px);
+  }
+
+  to {
+    opacity: 1;
+    transform: translateY(0px);
+  }
+}
```

Comment:
for simple animations like this, you can use a `transition` instead of an `animation` -- I'll show you how!